JavaScript-Files/test_ui.js

```diff
@@ -0,0 +1,32 @@
+function Player(name) {
+	this.name    = name,
+	this.money   = 15,
+	this.workers = 0;
```

Comment:
__[jshint]__
* Expected an assignment or function call and instead saw an expression.

<!-- Comment by [SideCI](https://sideci.com) -->